src/pages/home/sidebar/OptionRow.js

```diff
@@ -1,6 +1,6 @@
 import _ from 'underscore';
 import lodashGet from 'lodash/get';
-import React, {memo} from 'react';
```

Comment:
Revert this file and user 
  let touchableRef = null;
  
  as
  ```js
  const OptionRow = (props) => {
    let ref = null;
  ```
  
  then update 
  ```js
  <TouchableOpacity
                    ref={el => touchableRef = el}
                    onPress={(e) => {
                        e.preventDefault();
                        props.onSelectRow(props.option, touchableRef);
                    }}
    ```